src/webui/www/private/scripts/client.js

```diff
@@ -712,7 +712,7 @@ window.addEventListener("DOMContentLoaded", (event) => {
         trackerFilterList.appendChild(createLink(TRACKERS_ALL, "QBT_TR(All)QBT_TR[CONTEXT=TrackerFiltersList]", torrentsTable.getRowSize()));
         trackerFilterList.appendChild(createLink(TRACKERS_TRACKERLESS, "QBT_TR(Trackerless)QBT_TR[CONTEXT=TrackerFiltersList]", trackerlessTorrentsCount));
         trackerFilterList.appendChild(createLink(TRACKERS_ERROR, "Tracker error", trackerTorrentStatusesCount.error));
-        trackerFilterList.appendChild(createLink(TRACKERS_ANNOUNCE_ERROR, "Other error", trackerTorrentStatusesCount.announce_error));
+        trackerFilterList.appendChild(createLink(TRACKERS_ANNOUNCE_ERROR, "Announce error", trackerTorrentStatusesCount.announce_error));
```

Comment:
in the GUI this is shown as "other error" as shown
https://github.com/qbittorrent/qBittorrent/blob/5a4b3b25d3bb3379c3b579bcf805e8e0b7f2bc7a/src/gui/transferlistfilters/trackersfilterwidget.cpp#L103

my initial motivation was to keep consistency but in glassez PR he used the more semantic "announce error" because that's what it actually represents
https://github.com/qbittorrent/qBittorrent/compare/master...glassez:qBittorrent:tracker-status-webapi#diff-26f23205950f80a2659a1bc18ee3d7f4fc290408ae8784d0eb003bdfe36fd0bdR126

let me know if this incosistency is worth it